### PR TITLE
fix: apply plugins when preview.refresh() is invoked

### PR DESCRIPTION
**Pull request by dongwoo-kim**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
- fix: apply plugins when preview.refresh() is invoked
- refactor: remove highlight-js dependency from editor core

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: plugins/code-syntax-highlight/src/js/helper.js [hunk 1]
```diff
@@ -17,7 +17,13 @@ export function registerHljsToEditor(editor, hljs) {
     if (codeBlockLanguages.indexOf(type) < 0) {
       codeBlockLanguages.push(type);
     }
```
Comment (seonim-ryu) [inline plugins/code-syntax-highlight/src/js/helper.js]: 여긴 재성선임님이 작업하신 API로 변경하면 되겠네요ㅎ

File: plugins/code-syntax-highlight/src/js/helper.js [hunk 2]
```diff
@@ -17,7 +17,13 @@ export function registerHljsToEditor(editor, hljs) {
...
-  });
 
-  codeBlockManager.setHighlightJS(hljs);
+    const convertor = codeText => hljs.highlight(type, codeText).value;
+    const aliases = hljs.getLanguage(type).aliases || [];
```
Comment (seonim-ryu) [inline plugins/code-syntax-highlight/src/js/helper.js]: 이렇게는 안되나요?ㅎ
```js
const { aliases = [] } = hljs.getLanguage(type);
```

File: apps/editor/src/js/preview.js [hunk 3]
```diff
@@ -36,12 +37,37 @@ class Preview {
...
+    const contentEl = this._previewContent;
+    let targetEls;
+
+    if (mdNodeIds) {
+      targetEls = mdNodeIds
+        .map(id => contentEl.querySelector(`[data-nodeid="${id}"]`))
+        .filter(Boolean);
+    } else {
+      targetEls = [contentEl];
+    }
```
Comment (seonim-ryu) [inline apps/editor/src/js/preview.js]: 그냥 else 조건 값을 먼저 초기화해주면 안되나요??
```js
let targetEls = [contentEl];

if (mdNodeIds) {
  targetEls = mdNodeIds
    .map(id => contentEl.querySelector(`[data-nodeid="${id}"]`))
    .filter(Boolean);  
}
```

File: plugins/code-syntax-highlight/src/js/helper.js (hunk 2 above)
Comment (dongwoo-kim) [reply plugins/code-syntax-highlight/src/js/helper.js]: 오. 더 낫네요 ㅎ 수정하겠습니다.

File: apps/editor/src/js/preview.js (hunk 3 above)
Comment (dongwoo-kim) [reply apps/editor/src/js/preview.js]: 음. 더 이쁘긴 하네요 ㅎㅎ

**Pull request merged**